src/browsers.ts

```diff
@@ -40,7 +40,7 @@ export function normalize(
  *
  * If `names` is empty, or contains `all`, all installed browsers will be used.
  */
-export async function expand(names: string[]): Promise<wd.Capabilities[]> {
+export async function expand(names: string[], browsersOptions: {}): Promise<wd.Capabilities[]> {
```

Comment:
Can we get a better type annotation  for `browserOptions`, or even better use a `Map`?

Reply:
Possibly but I doint know how to program in typescript so not sure how to or to make it better.

Reply:
Well, we should have some description of this parameter. What's valid to pass to it?

Reply:
@justinfagnani the valid parameter for this would look like

      "browsersOptions": {
        "chrome": [
          "window-size=1920,1080",
          "headless",
          "disable-gpu"
        ],
        "firefox": [
          "--headless"
        ]
      }

or

      "browsersOptions": {
        "chrome": [
          "window-size=1920,1080",
          "headless",
          "disable-gpu"
        ]
      }

Reply:
Ok, looks like `browserOptions: {[name: string]: string[]}` or I think a `Map` is even better for map-like objects, so `Map<string, string[]>` if you make that change.

Reply:
Would it be browserOptions: Map<string, string[]>?

sorry for late reply.

Reply:
yeah, though that might not work because options are piped in from JSON. Let's just do `browserOptions: {[name: string]: string[]}`

Reply:
Thanks, done. :)